scripts/submit_emod3d.py

```diff
@@ -71,20 +72,21 @@ def write_sl_script(
         job_description="emod3d slurm script",
         additional_lines="#SBATCH --hint=nomultithread",
         target_host=machine,
+        write_directory=write_directory,
     )
 
     fname_slurm_script = "run_emod3d_%s_%s.sl" % (srf_name, const.timestamp)
+    fname_slurm_script = os.path.abspath(
```

Comment:
Less confusing to merge these two lines

Reply:
Why abspath?

Reply:
Returns the full path from root, so things like . and ~ are expanded